Server/mods/deathmatch/logic/lua/CLuaFunctionParseHelpers.cpp

```diff
@@ -271,6 +271,12 @@ ADD_ENUM(CAccountPassword::SHA256, "sha256")
 ADD_ENUM(CAccountPassword::MD5, "md5")
 IMPLEMENT_ENUM_END("account-password-type")
 
+IMPLEMENT_ENUM_CLASS_BEGIN(ESyncType)
+ADD_ENUM(ESyncType::BROADCAST, "broadcast")
+ADD_ENUM(ESyncType::LOCAL, "local")
+ADD_ENUM(ESyncType::SUBSCRIBE, "subscribe")
+IMPLEMENT_ENUM_CLASS_END("sync-type")

```

Comment:
```suggestion
IMPLEMENT_ENUM_CLASS_END("sync-mode")
```
we can keep it the way it is in the code, but i think it's clearer if we document this as a "sync mode"